src/main/java/com/thealgorithms/datastructures/trees/TreeRandomNode.java

```diff
@@ -17,33 +16,62 @@ the inOrder() method to store the values in the arraylist, then find the size of
   Step 6: STOP
  */
 
-
 import java.util.ArrayList;
 
-// Using auxiliary array to find the random node in a given binary tree
+/**
+ * Using auxiliary array to find the random node in a given binary tree
+ * @author : Suraj Kumar
+ * Github : https://github.com/skmodi649
+ */
 class Node {
     int item;
-    Node left, right;
+    NodeTree left, right;
+	public String data;
 
     public Node(int key) {
         item = key;
         left = right = null;
     }
+
+	public void insert(int value) {
```

Comment:
Why have you added those methods? They are empty and, thus, useless

Reply:
yes as i said someone wrote the calls to them but those called methods themselves werent declared anywhere. i just implemented them blanc just to have the full project compile without errors. i guess its up to the one who wrote the calls to finish their methods as he should know how he meant to implement them.

Reply:
Where are they called?